src/components/PaginatedThumbnails.js

```diff
@@ -13,6 +13,15 @@ export default class PaginatedThumbnails extends Component {
 		this.state = {
 			width: 500,
 		};
+
+		this.container = null;
+
+		this.setContainerRef = this.setContainerRef.bind(this);
+		this.updateWidth = this.updateWidth.bind(this);
+	}
+
+	setContainerRef (e) {
```

Comment:
Why not use `React.createRef ` https://reactjs.org/docs/react-api.html#reactcreateref ?

Reply:
it is not a function, I think it is because of react version

Reply:
`createRef` should work here.
Create it in the constructor (`this.container = createRef()`), reference it in the component (`<div ref={this.container}`), and use it where you need to (`this.setState({ width: this.container.current.offsetWidth });`)